dist/toolwindow.js

```diff
@@ -884,7 +912,8 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
         }
       },
       _suppressEvent: function _suppressEvent(evt) {
-        if (evt.stopPropagation) {
+		//this._log('_suppressEvent: '+(this._isDrag?' Drag':'')+(this._isResize?' resize=':'')+this._resizeMode+(this._isButton?' Button':''));		
```

Comment:
with `_log` being conditional, feel free to keep logging in (: Alternatively, if this logging isn't useful, please delete commented code.

Reply:
removed